contents/blog/session-recording-performance.md

```diff
@@ -0,0 +1,182 @@
+---
+date: 2022-08-31
+title: "Session Recording Demystified: Examining the (trivial) performance hit"
```

Comment:
Possible alternative headlines:

- Will session recording slow down your website? Examining the (trivial) impact